test/load/packer/consul-ami/scripts/move-files.sh

```diff
@@ -5,8 +5,8 @@ mv /home/ubuntu/scripts/conf.yaml /etc/datadog-agent/conf.d/consul.d/
 mv /home/ubuntu/scripts/datadog.yaml /etc/datadog-agent/
 
 ##Move Consul Config that hooks up to datadog
+mkdir -p /opt/consul/config
 mv /home/ubuntu/scripts/telemetry.json /opt/consul/config/
```

Comment:
If you're copying the file here it might make more sense to chown it right after to have the chown be as close to the file copy as possible. The chown in the userdata script on `/opt/consul/config/telemetry.json` seems independent of the rest of the userdata script and I wasn't sure where the `telemetry.json` file came from at first glance.

Reply:
I moved it mainly to allow Consul to make it's own User:Group during installation/set up.

Reply:
Hm I'm not sure what that means. The user:group isn't parameterized to go into the script so someone would have to modify that in the script prior to running tf.